torch/csrc/jit/codegen/cuda/lower_index.cpp

```diff
@@ -38,6 +38,13 @@ void IndexLowering::pushBack(Expr* expr) {
   }
 }
 
+Expr* IndexLowering::back() const {
+  if (active_scope_ == nullptr) {
+    return lowered_exprs_.back();
```

Comment:
Can you please add an assertion that `lower_exprs_` and `active_scope_->exprs()` are not empty? I understand they should not become empty, but debugging bugs due to `std::vector::front, std::vector:back` are not something always easy to diagnose.

Reply:
Updated. Thanks.